spec/routing/dishes_routing_spec.rb

```diff
@@ -0,0 +1,39 @@
+require "rails_helper"
+
+RSpec.describe DishesController, type: :routing do
+  describe "routing" do
+
+    it "routes to #index" do
+      expect(:get => "/dishes").to route_to("dishes#index")
```

Comment:
Plz try to use new hash syntax. (I guess this is auto generated code, right?) I would still update it.